app/models/item.rb

```diff
@@ -6,4 +6,8 @@ class Item < ApplicationRecord
   has_many :orders, through: :order_items
 
   enum item_status: ['available', 'retired']
+
+  def get_quantity_in_order(order)
```

Comment:
I'm pretty sure this returns the quantity for an item in an order-item. I'm okay with the name. other thoughts? @CjMoore @valerietrudell